### DISCUSSION TITLE
simpleStorage.sol

### DISCUSSION THREAD

**Discussion by Odinaka12** [smartcontractkit/full-blockchain-solidity-course-py#1673] [Q&A]
It keeps showing error in one line of code. 
Pls help me check if am doing it wrong 
![IMG_20220826_143937](https://user-images.githubusercontent.com/99669756/186919255-a876f2ab-9639-4b6e-a268-24388ba4e708.jpg)

**Comment by cromewar** [comment]
Hello @Odinaka12 
Please do not upload photos using a cellphone, take screen shot with `Win + Shift + S` on windows, and please give a check to #980 to reformulate your question.

Anyways, on the bottom left corner (Compiler tab) remix should show the error message, share it.
